setup.py

```diff
@@ -70,14 +70,16 @@ def _load_py_module(name: str, location: str) -> ModuleType:
 
 
 if __name__ == "__main__":
-    setup_tools = _load_py_module(name="setup_tools", location=os.path.join(".actions", "setup_tools.py"))
+    setup_tools = _load_py_module(name="setup_tools", location=os.path.join(_PATH_ROOT, ".actions", "setup_tools.py"))
+    assistant = _load_py_module(name="assistant", location=os.path.join(_PATH_ROOT, ".actions", "assistant.py"))
```

Comment:
(not for this PR) I think these 2 files could be merged. Is there any advantage to having them separated?